### DISCUSSION TITLE
GitHub Universal Code Search Indexing and Recently Indexed Issue

### DISCUSSION THREAD

**Discussion by charlieley** [community/community#36514] [Other Feature Feedback, Questions, & Ideas] [answered]
I had asked this on a previous occasion, but perhaps hadn't made it clear. In any case, I didn't get a response, and the same issue seems to be happening again. Would like to understand more about this.

I use universal code search on GitHub, primarily in this format, to research the usage pattern of certain technologies: 
https://github.com/search?o=desc&q=<query>&s=indexed&type=Code

The "s=indexed" filter allows the results to be sorted according to the latest which has been indexed by GitHub for whatever reason.

For more popular technologies, such as say, AWS, obtainable through:
https://github.com/search?o=desc&q=aws&s=indexed&type=Code
Here, the results are generally indexed by the minute and made available by the code search.

However, **on occasion, the code search doesn't show any recently indexed result**. For example, the code search described above for AWS:
https://github.com/search?o=desc&q=aws&s=indexed&type=Code
It shows results which have been indexed 3 days ago and not any newer ones, though doubtless, following the trends, there would have been many new files containing AWS in the meanwhile.

This has happened on previous occasions before, on one occasion, the indexing for newer results was out for 8-10 days I think.

I understand it's a big task for the search indexer to be up to date, however, it's something that has worked, and has been working fine generally, with intermittent interruptions, such as what is happening at the moment.

I'd like to ask if
 - there's an ongoing issue with the indexer, because of which newer files containing search queries aren't getting indexed? 
 - If so, is it something that is tracked and its status updated? I looked at [GitHub status](https://www.githubstatus.com/), but everything is normal, and no particular item jumps out as being related to the search as I have described.
 - When the indexer or Search starts indexing new files again, would the content previously generated in the meantime be also indexed?

I do not use cs.github.com, as I don't have an invite yet.

**Comment by pl0mo** [comment]
Same problem here.

**Reply by charlieley** [reply]
It's working again now, after 5 days.

It'd be good to understand why it happens and if it's monitored with some status update...

**Reply by pl0mo** [reply]
In mi case, the problem persist I will to try to log on with new web browser profile to discard cache or similar  issues.

Regard.

**Comment by ossanna16** [comment]
Hey there @charlieley and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by look** [comment, marked as answer]
@charlieley sorry about that. This has been addressed and the results are up-to-date now.

It's interesting to hear about your use case, because at the moment the code search Technology Preview does not support sorting by index date. We also reindex it frequently so such a sort would not be particularly useful.
